packages/amplify-category-predictions/commands/predictions/update.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
         context.print.success(`Successfully updated resource ${resourceName} locally`);
       })
       .catch(err => {
-        context.print.info(err.stack);
+        context.print.error(err.message);
```

Comment:
This is going to swallow errors on update there is a possibility of users losing information of their errors on update of a valid project